config.yaml

```diff
@@ -82,8 +82,9 @@ params:
   # This menu appears only if you have at least one [params.versions] set.
   version_menu: Versions
   versions:
-    latest: v3.5
+    latest: v3.6
```

Comment:
I think `latest` should stay `v3.5` until `v3.6` comes out of alpha, and is actually released. I'd suggest updating this the day of the release.

Reply:
Ok, sound good. Thanks @nate-double-u